editor.planx.uk/src/testUtils.tsx

```diff
@@ -28,12 +19,21 @@ export const setup = (
   ...render(<ThemeProvider theme={defaultTheme}>{jsx}</ThemeProvider>),
 });
 
+// export const axe = configureAxe({
+//   rules: {
+//     // Currently, jest-axe does not correctly evaluate this rule due to an issue with jsdom
+//     // https://github.com/dequelabs/axe-core/issues/2587
+//     // To pass this test, non-decorative MUI icons should always use the 'titleAccess' prop
+//     "svg-img-alt": { enabled: false },
+//   },
+// });
+
 /**
  * Mock the MUI Fade component used in @planx/components/shared/Preview/Card.tsx
  * Required as this frequently updates following the final "expect()" call of a test,
  * leading to multiple "an update was not wrapped in act(...)" warnings
  * Docs: https://testing-library.com/docs/example-react-transition-group/
  */
-export const mockFade = jest.mock("@mui/material/Fade", () =>
-  jest.fn(({ children }: FadeProps) => <div>{children}</div>),
-);
+// export const mockFade = vi.mock("@mui/material/Fade", () =>
+//   vi.fn(({ children }: FadeProps) => <div>{children}</div>),
+// );
```

Comment:
This looks like a mock we might want to / need to migrate still?

Reply:
Just pushed up [f32e69c](https://github.com/theopensystemslab/planx-new/pull/3544/commits/f32e69cf4f8528057bb6c17fde92407e2e16a1af) which addresses this.

`mockFade` has had to move to the test setup file as vitest does not allow mocks to be exported.